playbook/app/pb_kits/playbook/pb_advanced_table/table_body.rb

```diff
@@ -16,6 +16,11 @@ class TableBody < Playbook::KitBase
                             default: []
       prop :collapsible_trail, type: Playbook::Props::Boolean,
                                default: true
+      prop :responsive, type: Playbook::Props::Enum,
+                        values: %w[none scroll],
+                        default: "none"
```

Comment:
Hey! So default for the responsive prop should be the same for advanced_table, body and header, you'll have to adjust your logic, but these should be aligned in case a subcomponent structure is used!